pkg/meta/version.go

```diff
@@ -2,7 +2,7 @@ package meta
 
 const (
 	// InstanceManagerAPIVersion is used for compatibility check for longhorn-manager
-	InstanceManagerAPIVersion    = 5
+	InstanceManagerAPIVersion    = 6
```

Comment:
Although we add new methods to `InstanceService` gRPC interface, it does not really need to bump `InstanceManagerAPIVersion` here, because:

- When an old client calls any existing methods in a new instance manager w/ these new methods, everything should work correctly because of backward compatibility in gRPC.
- When a new client calls any new method in an old instance manager, even though the method does not exist, the call will just do nothing, because of forward compatibility.

We Need to review the current traditional version flag way to control how new/old longhorn manager communicates with new/old instance manager, but rather rely on gRPC capabilities.

ref: https://github.com/longhorn/longhorn/issues/4833